mne/io/nirx/nirx.py

```diff
@@ -19,15 +19,25 @@
 
 
 @fill_doc
-def read_raw_nirx(fname, preload=False, verbose=None):
+def read_raw_nirx(fname, nosatflags=False, preload=False, verbose=None):
     """Reader for a NIRX fNIRS recording.
 
-    This function has only been tested with NIRScout devices.
+    This function has only been tested with NIRScout and NIRSport1 devices.
 
     Parameters
     ----------
     fname : str
         Path to the NIRX data folder or header file.
+    nosatflags : bool
+        (Only relevant for NIRSport devices). Decide to use *.nosatflags_wlX
+        instead of standard *.wlX files.
+        The NIRSport probes can saturate during the experiment. Starting with
+        NIRStar 14.2, those saturated values are replace with 'NaN' in the
+        standard *.wlX files. The probed values are stored in another file
```

Comment:
```suggestion
        standard *.wlX files. The measured values are stored in another file
```

Reply:
The term probe is commonly used in NIRS literature to mean the location of optodes (montage).